apps/ewallet/lib/ewallet/web/v1/serializers/api_key_serializer.ex

```diff
@@ -32,8 +32,8 @@ defmodule EWallet.Web.V1.APIKeySerializer do
       id: api_key.id,
       name: api_key.name,
       key: api_key.key,
-      account_id: api_key.account.id,
```

Comment:
Nope, can't do that yet 😅 We'll remove the field in 2.0 when we make a new API version.